caas/kubernetes/provider/services.go

```diff
@@ -0,0 +1,16 @@
+// Copyright 2019 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package provider
+
+import (
+	core "k8s.io/api/core/v1"
+)
+
+var preferredControllerServiceTypes = map[string]core.ServiceType{
```

Comment:
I think this can all go in caas/metadata.go as that's where we have started recording cluster specific preferences like storage etc

Reply:
moved to metadata.go